features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/about/AboutEvents.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.features.preferences.impl.about
+
+sealed interface AboutEvents {
+    object MyEvent : AboutEvents
```

Comment:
Perhaps unwanted file?

Reply:
Yes, I did not really want to delete the (generated) files because, when we will need it, we will have to create theme again, but I have done the cleanup in 5dd0a1b477ed6c02f18155a8291a3b7c8b046fed